README.md

```diff
@@ -1,2 +1,38 @@
-# use-is-mounted-ref
-Returns a ref with a current value of true if mounted
+# useIsMountedRef
+Returns a `current` value of true when a component is mounted.
+
+## Motivation
+Useful when working with server side rendering. Comparable to `componentDidMount` in a class-based component. Does __not__ trigger a rerender.
```

Comment:
Waar hij ook veel voor wordt gebruikt is het checken of een component nog gemount is na een asynchrone actie. Misschien goed om hier te vermelden.